File: app/src/main/java/com/tes/aop/ApiService.java
package com.tes.aop;

import retrofit2.Call;
import retrofit2.http.*;
import java.util.List;

/**
 * ApiService — Define your API endpoints here.
 * TODO: Replace or add methods to match your API
 */
public interface ApiService {

    @GET("items")
    Call<List<String>> getItems();

    // Contoh POST:
    // @POST("items")
    // Call<String> createItem(@Body String item);
}

File: app/src/main/java/com/tes/kk/ApiService.kt
package com.tes.kk

import retrofit2.Call
import retrofit2.http.*

/**
 * ApiService — Define your API endpoints here.
 * TODO: Replace or add methods to match your API
 */
interface ApiService {

    @GET("items")
    fun getItems(): Call<List<String>>

    // Contoh POST:
    // @POST("items")
    // fun createItem(@Body item: String): Call<String>
}
